gen/field.go

```diff
@@ -875,3 +900,23 @@ func verifyUniqueFieldLabels(fs compile.FieldGroup) error {
 	}
 	return nil
 }
+
+// RedactedLabel provides a mechanism to redact certain struct fields from
+// the outputs of String(), Error() and MarshalLogObject() methods.
+//
+//	struct Contact {
+//	    1: required string name
+//	    2: required string email (go.redacted)
+//	}
+const RedactedLabel = "go.redacted"
```

Comment:
could we change the label to `go.redact` this will be in the verb from like 'log' in the `go.nolog` annotation.